tests/CMakeLists.txt

```diff
@@ -1,98 +1,35 @@
-include_directories(
-../src
- ../src/games
- ../src/games/weakschur
- ../src/third_party
- ../src/third_party/fmt/include
- ../src/tube/src_cpp
- ../src/third_party/pybind11/include/
- )
+include(FetchContent)
+FetchContent_Declare(
+  googletest
+  # Specify the commit you depend on and update it regularly.
+  URL https://github.com/google/googletest/archive/5376968f6948923e2411081fd9372e71a59d8e77.zip
+)
+# For Windows: Prevent overriding the parent project's compiler/linker settings
```

Comment:
Windows!!!

Reply:
Haha, I copy-pasted it from another file